### DISCUSSION TITLE
[Projects Beta] Quickly create issue without creating draft

### DISCUSSION THREAD

**Discussion by ptrwtts** [community/community#7784] [Projects and Issues]
I would like a way to quickly create an issue from the project view without needing to first create a draft, convert to issue (pick repo), then add fields (e.g. assignee). 

Is this possible and I'm missing it? My expectation is that i would use shortcuts like `repo:` and `assignee:` while typing the issue name, in order to do it all in one step.

**Comment by ptrwtts** [comment]
The new Draft editing UI is 🔥 and 95% solves my issue. The one thing missing for me is the ability to assign someone when converting to an issue. That way, I could stay in the project view, and never need to open issues, other than to make edits.

**Comment by V-iktor** [comment]
I think you should be able to set a default repo and turn off this "draft" feature. 

Who wants to create drafts anyway? If I dont want the issue I would just delete it.
